source/docs/v3/integrations/sail.blade.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Laravel Sail integration
+extends: _layouts.documentation
+section: content
+---
+
+# Laravel Sail {#sail}
+
+## Creating tenants
+
+Before creating any tenants you have to ensure that the `sail` user has the necessary permissions.
```

Comment:
`the necessary permissions` doesn't make sense here, it should say specifically which permissions. Before telling people what to change, we should tell them why they are doing it — i.e. what permissions are missing.

Reply:
I have re-framed the statement. Please check how it sounds.